apps/ledger-live-desktop/tests/utils/dummy-live-app/src/index.css

```diff
@@ -2,3 +2,7 @@ body {
   margin: 0;
   font-family: system-ui, "Segoe UI", "Roboto", sans-serif;
 }
+
+*::-webkit-scrollbar {
+  display: none
```

Comment:
Missing semicolon :D

Reply:
haha good catch, but technically the final semicolon in a set of statements is optional

Reply:
I'll add it anyway 😉